libcontainer/container_linux.go

```diff
@@ -758,6 +773,19 @@ func (c *linuxContainer) addMaskPaths(req *criurpc.CriuReq) error {
 		}
 		req.Opts.ExtMnt = append(req.Opts.ExtMnt, extMnt)
 	}
+	return nil
+}
+
+func waitForCriuLazyServer(r *os.File, status string) error {
+
+	data := make([]byte, 1)
+	count, _ := r.Read(data)
```

Comment:
Can you add error handling for various calls in this function?  Read, OpenFile, and Write are all missing